### PR TITLE
Added: Toast component

### PR DESCRIPTION
**Pull request by pratik-kumar-sahu**
- Added Toast component.
- Live link --> https://deploy-preview-13--fuzz-ui.netlify.app/pages/docs.html

### CODE REVIEW

**Review by JatuFaique — COMMENTED** [review]
Great work, looks good to me.

**Comment by parthpandyappp** [comment]
Looks Good to me. Go ahead with merge.

File: styles/_components.css [hunk 1]
```diff
@@ -94,3 +93,71 @@
...
+    background-color: #2b3e50;
+    overflow: auto;
+    border-radius: 1rem;
+}
+
+.iframe {
+    width: 100%;
+    height: auto;
+    overflow: hidden;
+    /* border: 1px solid var(--color-background); */
```
Comment (AtulHande03a) [inline styles/_components.css]: remove the comments if not needed

**Review by AtulHande03a — APPROVED** [review]

File: styles/_components.css (hunk 1 above)
Comment (pratik-kumar-sahu) [reply styles/_components.css]: Sure, in final merge I'll remove all the commented codes

**Pull request merged**